packages/core/ios/RNEmbrace/EmbraceManager.swift

```diff
@@ -581,4 +594,74 @@ class EmbraceManager: NSObject {
 
         resolve(true)
     }
+    
+    @objc
+    func logNetworkRequest(
+        _ url: String,
+        httpMethod: String,
+        startInMillis: Double,
+        endInMillis: Double,
+        bytesSent: Double,
+        bytesReceived: Double,
+        statusCode: Double,
+        error: String,
+        resolver resolve: RCTPromiseResolveBlock,
+        rejecter reject: RCTPromiseRejectBlock
+    ) {
+        if Embrace.client == nil {
+            reject("RECORD_LOG_NETWORK_REQUEST_ERROR", "Error recording span, Embrace SDK may not be initialized", nil)
+            return
+        }
+
+        let attributes = attributeStringsFrom(dict: [
+            "http.response.status_code": String(Int(statusCode)),
```

Comment:
for `bytesSent`, `bytesReceived`, `statusCode` should they be included if they are passed in as -1? How does Android handle that?

Reply:
done